Resources/public/js/views/tabs/ez-locationviewversionstabview.js

```diff
@@ -156,15 +180,90 @@ YUI.add('ez-locationviewversionstabview', function (Y) {
         },
 
         /**
+         * Deletes one or several drafts
+         *
+         * @method _deleteDraft
+         * @protected
+         */
+        _deleteDraft: function () {
+            var c = this.get('container'),
+                versions = [];
+
+            versions = Y.Array.reject(this.get('versions').DRAFT, function (version) {
+                var checkbox = c.one(
+                    '.ez-draft-version-checkbox[data-version-id="' + version.get('id') + '"]'
+                );
+
+                if (checkbox && checkbox.get('checked')) {
+                    return false;
+                }
+                return true;
+            });
+
+            if (versions.length > 0) {
+                this._disableVersionsCheckboxes();
+
+                /**
+                 * Fired when the user clicks on the Delete draft button
+                 *
+                 * @event deleteVersionDraft
+                 * @param {Array} of eZ.VersionInfo
+                 *        {Function} afterDeleteVersionsCallback
+                 *
+                 */
+                this.fire('deleteVersionDraft', {
+                    versions: versions,
+                    afterDeleteVersionsCallback: Y.bind(this._afterDeleteVersionsCallback, this)
+                });
+            }
+        },
+
+        /**
+         * Callback function called after removing version(s).
+         *
+         * @method _afterDeleteVersionsCallback
+         * @protected
+         * @param {Boolean} versionsRemoved if TRUE the view is reloaded, if FALSE it just enables checkboxes
+         */
+        _afterDeleteVersionsCallback: function (versionsRemoved) {
+            if (versionsRemoved) {
```

Comment:
I would always refresh the version list even in case of error, so you can completely remove that method and use `this._refresh` as the `afterDeleteVersionsCallback`